docs/syntax/scss.md

```diff
@@ -278,7 +278,23 @@ Modules that make use of styles defined in other modules that support silent mod
 		@include oAnotherThingFoo();
 	}
 
-Finally, in documentation, modules *must* provide information about both silent and non-silent methods, where supported, and must put the default first (i.e. if silent mode is by default **on**, the module must document the silent mode integration first).
+When a module that supports silent mode is used with silent mode turned off (i.e. `$o-{modulename}-is-silent: false`), after the module has output the CSS, it *must* also reset the value of `$o-{modulename}-is-silent` to true, (this prevents the accidental output of styles if the module is included a second time):
```

Comment:
'is used' where?  In another module?

I'm not sure which modules you're talking about here.  Maybe using 'dependent' and 'dependency' or A and B would help.

Finally in principle I'm unsure about whether one module unsilencing another module is a good idea.  Don't we have a rule that modules can only be used as dependencies of other modules if they are used in silent mode?